### ISSUE TITLE
Search input violates WCAG 1.3.1, WCAG 3.3.2

### ISSUE THREAD

**Issue by auchenberg** [microsoft/vscode-docs#2616]
label: Form elements must have labels (1) WCAG 1.3.1, WCAG 3.3.2

Url: https://code.visualstudio.com/docs/editor/versioncontrol

Links:
- https://dequeuniversity.com/rules/axe/3.2/label?application=msftAI
- https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships
- https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html

Code path:
```
<input type="text" name="q" class="search-box form-control" placeholder="Search Docs">
--



```

**Comment by Nikhitha-13** [comment]
**NOTE:**
The issue is observed through out the application in the following URLs
1. code.visualstudio.com/docs
2. code.visualstudio.com/download.
3. code.visualstudio.com/docs/supporting/faq.
4. code.visualstudio.com/docs/setup/linux.
5. code.visualstudio.com/docs/getstarted/settings.
6. code.visualstudio.com/docs/python/python-tutorial.
7. code.visualstudio.com/updates/v1_30
8. code.visualstudio.com/docs/editor/whyvscode.
9. code.visualstudio.com/docs/getstarted/tips-and-tricks
10. code.visualstudio.com/updates/v1_29
11. code.visualstudio.com/docs/editor/integrated-terminal
12. code.visualstudio.com/docs/editor/debugging
13. code.visualstudio.com/docs/editor/versioncontrol

**Comment by Nikhitha-13** [comment]
![Bug_#2616-issue](https://user-images.githubusercontent.com/50437925/57451226-4fb1aa00-727e-11e9-9221-ea5531c5b4e9.png)
![Bug_2616_issue](https://user-images.githubusercontent.com/50437925/57451262-6952f180-727e-11e9-8a79-1457cb2e12b5.png)
![Bug_#2616_issue](https://user-images.githubusercontent.com/50437925/57451283-766fe080-727e-11e9-94b0-e4477c952333.png)
![Bug_2616_issue](https://user-images.githubusercontent.com/50437925/57451310-8e476480-727e-11e9-8c34-abd0056c51ab.png)
![Bug_#2616_issue](https://user-images.githubusercontent.com/50437925/57451332-9acbbd00-727e-11e9-8a3d-7bd1c9a4d93b.png)
![Bug_#2616_issue](https://user-images.githubusercontent.com/50437925/57451371-acad6000-727e-11e9-8194-bbce3139fed6.png)
![Bug_#2616_issue](https://user-images.githubusercontent.com/50437925/57451400-bb941280-727e-11e9-9c28-7f7cf5680f8b.png)
![Bug_#2616_issue](https://user-images.githubusercontent.com/50437925/57451420-d1093c80-727e-11e9-9c20-138b7fd273d0.png)
![Bug_#2616_issue](https://user-images.githubusercontent.com/50437925/57451450-e8e0c080-727e-11e9-8e3a-d177ed5cb00c.png)
![Bug_#2616_issue](https://user-images.githubusercontent.com/50437925/57451467-fac26380-727e-11e9-8010-3c57cf93e6e6.png)
![Bug_#2616_issue](https://user-images.githubusercontent.com/50437925/57451492-0a41ac80-727f-11e9-8b3d-ed9391b43b32.png)
![Bug_#2616_issue](https://user-images.githubusercontent.com/50437925/57451507-188fc880-727f-11e9-8c4c-8a43bcab3a24.png)
![Bug_#2616_issue](https://user-images.githubusercontent.com/50437925/57451524-28a7a800-727f-11e9-9325-966e610074eb.png)

**Comment by gregvanl** [comment]
@Nikhitha-13 Fixed

**Comment by Nikhitha-13** [comment]
Checked the issue again. The issue is fixed.

**Comment by fsteffi** [comment]
GitHubTags:#CAITestPass_June’19;
